packages/db-objects-redis/src/lib/objects/objectsInRedisClient.ts

```diff
@@ -4431,6 +4435,19 @@ export class ObjectsInRedisClient {
         return result;
     }
 
+    // getObjectList is called without options
+    getObjectList(
+        params: ioBroker.GetObjectListParams,
+        callback?: ioBroker.GetObjectListCallback
+    ): void | Promise<ioBroker.CallbackReturnTypeOf<ioBroker.GetObjectListCallback>>;
```

Comment:
maybe distinguish the return type here (promise or not) depending on the callback arg like you did for _subscribe?